python/ray/actor.py

```diff
@@ -1545,6 +1545,15 @@ def _actor_method_call(
         return object_refs
 
     def __getattr__(self, item):
+        try:
+            self.__getattribute__("_ray_is_cross_language")
```

Comment:
what is `__getattribute__`? have never seen this before

is it equivalent to `getattr(self, "_ray_is_cross_language")`?

Reply:
No, they are different. The built-in function `getattr` calls `__getattribute__` first, and only if the attribute doesn't exist does it call `__getattr__`.

Reply:
is `__getattribute__` a stable public API?

Reply:
I couldn't find a reference that explicitly states it's a stable public API, but I believe it is. It is publicly documented with examples from Python documentation ranging from version 2.6 to 3.14.

* https://docs.python.org/3.14/reference/datamodel.html#object.__getattribute__
* https://docs.python.org/2.6/reference/datamodel.html#object.__getattribute__